additional_rule_metadata.json

```diff
@@ -617,6 +632,22 @@
         "recommendation": "false,",
         "commonWeaknessEnumeration": "398, 710"
     },
+    "unnecessary-bind": {
+        "issueClass": "Non-SDL",
+        "issueType": "Warning",
+        "severity": "Moderate",
```

Comment:
`severity` and `commonWeaknessEnumeration` differs from initial [rule metadata](https://github.com/Microsoft/tslint-microsoft-contrib/blob/9b25025c4912de65ca869c93a37c8bec84c94111/src/noUnnecessaryBindRule.ts#L17-L20)

Is it intentional change?